stl/inc/bitset

```diff
@@ -271,15 +290,15 @@ public:
         return *this;
     }
 
-    bitset& flip(size_t _Pos) { // flip bit at _Pos
+    _CONSTEXPR23 bitset& flip(size_t _Pos) { // flip bit at _Pos
         if (_Bits <= _Pos) {
             _Xran(); // _Pos off end
         }
 
         return _Flip_unchecked(_Pos);
     }
 
-    _NODISCARD unsigned long to_ulong() const {
+    _NODISCARD _CONSTEXPR23 unsigned long to_ulong() const {

```

Comment:
I note that this could be `noexcept(_Bits <= 32)` /* strengthened */`, but I'm not sure it's worth the trouble. (No change requested.) (Line 326 is similar.)

Anyone see motivation for this that I don't?